src/components/Selector/Selector.tsx

```diff
@@ -46,7 +46,7 @@ export function Selector<
           aria-invalid={!!get(errors, valuePath)?.message}
           aria-disabled={isDisabled}
           as="button"
-          className={`${button} ${styles.selectorButton} peer-focus:border-gray-d1 peer-focus:dark:border-blue-l2`}
+          className={`${button} ${styles.selectorButton}`}
```

Comment:
When a form containing a `Selector` was submitted and validation errors occur, the screen scrolls up to the first field with an error and the UI is as the following:
- with these classes
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/ca60c401-204b-42da-994c-b4899455cf5c)
- without:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/1b0b2f6e-d16b-4cc4-8d4e-4be0aa84fd86)

@ap-justin is it necessary to keep these classes present for `Selector`?

Reply:
I think it's necessary to distinguish which one of the invalid fields are currently in focus - more evident if user then tabs to fill out other field

Reply:
That makes sense :thinking:
A box shadow indicator might be a good indicator of a focused field while being more appealing to the eye (see image below), what do you think @ap-justin @SovereignAndrey ?
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/122a3d11-3eb5-4cd7-9a71-d3ac1fadfc3e)